common/persistence/cassandra/cassandraQueue.go

```diff
@@ -115,29 +117,43 @@ func (q *cassandraQueue) createQueueMetadataEntryIfNotExist() error {
 func (q *cassandraQueue) EnqueueMessage(
 	messagePayload []byte,
 ) error {
-	nextMessageID, err := q.getNextMessageID()
+	nextMessageID, err := q.getLastMessageID(q.queueType)
```

Comment:
rename nextMessageID to lastMessageID now that you have changed the method name

Reply:
renamed